config/ext/rails.yml

```diff
@@ -750,6 +805,16 @@ Rails/ReadWriteAttribute:
   Include:
     - app/models/**/*.rb
 
+Rails/RedundantActiveRecordAllMethod:
+  Description: Detect redundant `all` used as a receiver for Active Record query methods.
+  StyleGuide: 'https://rails.rubystyle.guide/#redundant-all'
+  Enabled: false # While we agree that User.all.find is redundant, it can be useful to make relation access more explicit for other query methods.
```

Comment:
Which cases do you have in mind where using an explicit `.all` is helpful?

Reply:
In an authorization context, something like `updatable_users = User.all.where(soft_deleted: false)` has a redundant `all` but could be more explicit and thus helpful to express "this affects virtually all users".

Don't have a strong opinion on it, but honestly I also don't see any real benefit from this cop. :man_shrugging:

Reply:
Don't have a strong opinion on it either. Inclined to enable it, but may as well stay disabled.

Reply:
Decision: we keep it disabled